assets/js/atomic/blocks/product/button/style.scss

```diff
@@ -0,0 +1,26 @@
+.wc-block-layout .wc-block-components-product-button {
```

Comment:
This is an additional layer of specificity which applies to all atomic blocks. There were theme compat issues without it.

Reply:
At the same time, this seems to be overriding some styles themes added on top of our blocks. A theme I use for testing is [Orchid Store](https://themebeez.com/themes/orchid-store/).

In `master`, the on sale badges were red:
![imatge](https://user-images.githubusercontent.com/3616980/83765976-1cf20500-a67c-11ea-8b66-74edf34a8b79.png)

But in this branch they became white because our selectors got more specificity and now their theming is lost:
![imatge](https://user-images.githubusercontent.com/3616980/83766131-4f9bfd80-a67c-11ea-9b5f-5561cd3d441d.png)

> There were theme compat issues without it.

Which themes were the ones with issues? If they were from the 20xx family, could we target them specifically?